lib/MassEdge/WordPress/Plugin/ExportMediaLibrary/Module/AdminPageExport.php

```diff
@@ -105,7 +105,7 @@ function page() {
         echo ob_get_clean();
     }
 
-    function export($exportName, $folderStructure, $compress) {
+    function export($exportName, $folderStructure, $compress, $overrides = []) {
```

Comment:
This function needs to be exposed at the plugin level, so other plugins can access it more easily.